external/nmea/parse.c

```diff
@@ -390,7 +390,9 @@ int nmea_parse_GPGSV( const char *buff, int buff_sz, nmeaGPGSV *pack )
     return 0;
   }
 
-  if ( type != 'P' && type != 'N' )
+  if ( type != 'P' && type != 'N' && type != 'L' && type != 'A' && type != 'B' )
+  //if ( type != 'P' && type != 'N' && type != 'L' && type != 'A' && type != 'B' && type != 'Q' )
```

Comment:
I think you could remove the lines you have commented out (here and elsewhere).

Reply:
OK fatto